package/correct_live_for_reboot

```diff
@@ -0,0 +1,34 @@
+#! /bin/sh
+
+# this script is called from the live installer once it finished copying the live
+# image. So we only do the minimal changes necessary to reboot into the system
+
+is_usb=0
+if test "$1" = usb; then
+  is_usb=1
+fi
+
+chkconfig sshd on
+chkconfig cron on
+if test -e /etc/init.d/boot.compcache; then
+   chkconfig boot.compcache off
+fi
+rm /usr/lib/systemd/system/sysinit.target.wants/langset.service
+rm /usr/lib/systemd/system/langset.service /usr/lib/systemd/system/sysinit.target.wants/langset.service
+
+
+: > /var/run/utmp
+
+# remove live-cd user
+userdel linux
+# a lot of stuff!
+rm -rf /home/linux/.local/share/akonadi
+mv /home/linux /home/linux~
+
+#undo journald.conf changes, bug 950999
+sed -i -e s@Storage=volatile@@ /etc/systemd/journald.conf
```

Comment:
in general I have impression that it make sense to create client for live installer doing it. Also I think each action should have documented rationale why it is needed.